File: swagger/models/get_custom_client_fields_response.py
from .base_model import BaseModel

from .pagination_response import PaginationResponse

class GetCustomClientFieldsResponse(BaseModel):
	"""
	no description available

	Attributes:

		pagination_response
		Contains information about the pagination used.

		custom_client_fields
		Contains information about the available custom client fields.
	"""

	input_map = {
		'PaginationResponse': ('pagination_response', PaginationResponse),
		'CustomClientFields': 'custom_client_fields',
		}

	output_map = {
		'pagination_response': 'PaginationResponse',
		'custom_client_fields': 'CustomClientFields',
		}

	pagination_response = None
	custom_client_fields = None

File: swagger/Models/GetCustomClientFieldsResponse.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class GetCustomClientFieldsResponse extends BaseModel
{
	/**
	 * Contains information about the pagination used.
	 * 
	 * @var PaginationResponse 
	 */
	public $paginationResponse;
	/**
	 * Contains information about the available custom client fields.
	 * 
	 * @var CustomClientField[] 
	 */
	public $customClientFields;

	protected function getInputMap()
	{
		return [
			'PaginationResponse' => ['paginationResponse', PaginationResponse::class, null],
			'CustomClientFields' => ['customClientFields', 'array', CustomClientField::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'paginationResponse' => ['PaginationResponse', PaginationResponse::class, null],
			'customClientFields' => ['CustomClientFields', 'array', CustomClientField::class],
			];
	}
}
